quickjs.c

```diff
@@ -47203,15 +47196,14 @@ static JSValue js_date_Symbol_toPrimitive(JSContext *ctx, JSValue this_val,
 static JSValue js_date_getTimezoneOffset(JSContext *ctx, JSValue this_val,
                                          int argc, JSValue *argv)
 {
-    // getTimezoneOffset()
     double v;
 
     if (JS_ThisTimeValue(ctx, &v, this_val))
         return JS_EXCEPTION;
     if (isnan(v))
         return JS_NAN;
     else
-        return JS_NewInt64(ctx, getTimezoneOffset((int64_t)trunc(v)));
+        return JS_NewInt64(ctx, getTimezoneOffset());
```

Comment:
I don't think it's correct to ignore the Date object's time value, per https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-date.prototype.gettimezoneoffset

Instead of using the current time you probably have to convert to UNIX timestamp and pass that to localtime/gmtime, otherwise it can produce the wrong results for dates in the past or future.